bigtable/google/cloud/bigtable/row.py

```diff
@@ -578,15 +597,15 @@ def commit(self):
             )
 
         data_client = self._table._instance._client.table_data_client
-        resp = data_client.check_and_mutate_row(
+        data_client.check_and_mutate_row(
             table_name=self._table.name,
             row_key=self._row_key,
             predicate_filter=self._filter.to_pb(),
             true_mutations=true_mutations,
             false_mutations=false_mutations,
         )
         self.clear()
-        return resp.predicate_matched
+        return self.row_content
```

Comment:
This changes the existing behavior, and doesn't match the service's results.  This actually will reduce usage and performance.

Reply:
I knew, it's an extra call, but reading row by indexed key looks lightly (if number/size of columns is not too large, of course). And changing the behavior was requested in issue. Is there are any indicator to understand, that issue approved by people, who know the details? It's not first time, when I PR'ed too fast. For newbie like me some things are not obvious (especially, when googlers said that it's not bad idea)